lineage/plotTree.py

```diff
@@ -5,11 +5,11 @@
 import networkx as nx
 
 cs = ['lightblue', 'orange', 'lightgreen', 'red', 'purple', 'olive', 'gray']
-stateColors = ['blue', 'orange', 'green', 'red', 'purple', 'olive', 'gray']
+stateColors = ['blue', 'green', 'orange', 'red', 'purple', 'olive', 'gray']
```

Comment:
Could you return this back to the original now that you created figure?

Reply:
> Wait, In figureS8-S10 the y_label and title are the same. THey should not be. Please edit this.
> 
> ![image](https://user-images.githubusercontent.com/34812750/140402367-33277f83-ab2c-4ea9-80f6-e057561b1890.png)

Sure thing. Just confirming, the y-axis label should be "Accuracy [%]" , correct?